pkgs/development/python-modules/sphinx-multiversion/default.nix

```diff
@@ -24,6 +24,8 @@ buildPythonPackage rec {
   dependencies = [ sphinx ];
   nativeCheckInputs = [ unittestCheckHook ];
 
+  doCheck = false; # because of "NO TESTS RAN" error
```

Comment:
```suggestion
```

Reply:
At least by looking at the log messages it seems like the imports check is still run, although the unit tests are disabled. is that just a misleading print in the output, or is that possible? @wegank

Reply:
I think the imports check is controlled by `dontUsePythonImportsCheck` instead.

Reply:
@wegank fixed